ext/strscan/strscan.c

```diff
@@ -524,7 +524,7 @@ strscan_get_pos(VALUE self)
     struct strscanner *p;
 
     GET_SCANNER(self, p);
-    return INT2FIX(p->curr);
+    return LONG2FIX(p->curr);
```

Comment:
Do we need to use `LONG2NUM()` not `LONG2FIX()`...?

```suggestion
    return LONG2NUM(p->curr);
```

Reply:
Corrected thanks.